Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1781,6 +1783,32 @@ namespace PlayerCommands
 				if (curr_item == item)
 				{
 					i->second.price = (float)money;
+					SendMarketGoodUpdated(base, i->first, i->second);
+					base->Save();
+
+					int page = ((curr_item + 39) / 40);

```

Comment:
While it's clear what these do, avoid magic numbers. Use constexpr variables to provide names to compile time constants.